src/interaction/selection.ts

```diff
@@ -18,15 +18,24 @@ export function start(): void {
 function recompute(): void {
     let s = document.getSelection();
 
-    if (selection !== null && data.start !== null && data.start !== getId(s.anchorNode)) {
+    // Bail out if we don't have a valid selection
+    if (s === null) { return; }
+
+    let anchorNode = getId(s.anchorNode);
+    let focusNode = getId(s.focusNode);
+
+    // Bail out if we got valid selection but not valid nodes
+    if (anchorNode === null && focusNode === null) { return; }
```

Comment:
>null [](start = 23, length = 4)

How could that happen?

Reply:
Added comment - basically, selection change event fires in Edge even on right click or click. If there was no previous selection, document.getSelection() returns null nodes.